packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/apple/appidpinokta.md

```diff
@@ -1,10 +1,12 @@
 In the **General Settings** section:
 
-* **Name** &mdash; Enter the name that you would expect to see on a button, such as **Sign in with Apple**.
+* **Name** &mdash; Enter a name for the Identity Provider configuration.
 * **Client ID** &mdash; Paste the Services ID that you obtained from Apple when you configured the Identity Provider in the <GuideLink link="../create-an-app-at-idp">previous section</GuideLink>. The Services ID identifies the particular instance of your app and is used as the OAuth `client_id`.
 * **Client Secret Signing Key** &mdash; Click **Browse files** and locate the key file (.p8 format) that you downloaded from Apple when you created the private key for client authentcation in the <GuideLink link="../create-an-app-at-idp">previous section</GuideLink>.
 * **Key ID** &mdash; Enter the Key ID that you obtained from Apple when you created the private key for client authentication in the <GuideLink link="../create-an-app-at-idp">previous section</GuideLink>. The Key ID represents the private key generated by Apple.
-* **Team ID** &mdash; Enter your Apple Team ID. This value must be the 10 character (alphanumeric) Team ID from your Apple developer account. This ID can be found in the upper-right corner of your Apple developer dashboard just below your account name.
+* **Team ID** &mdash; Enter your Apple Team ID. This value must be the alphanumeric Team ID from your Apple developer account. You can find this ID in the upper-right corner of your Apple developer dashboard just below your account name.
 * **Scopes** &mdash; Leave the defaults. These scopes are included when Okta makes an OpenID Connect request to Apple. See the [scope page](https://developer.apple.com/documentation/sign_in_with_apple/clientconfigi/3230955-scope) at Apple for more information on their required scopes.
 
-    > **Note:** By default, Okta requires the `email` attribute for a user. The `email` scope is required to create and link the user to Okta's Universal Directory.
\ No newline at end of file
+    By default, Okta requires the `email` attribute for a user. The `email` scope is required to create and link the user to Okta's Universal Directory. If your Identity Provider doesn't support this attribute, you can make it optional. See [Manage User Profiles](https://help.okta.com/en/prod/Content/Topics/Directory/eu-profile-editor.htm). For Just In Time (JIT) provisioning, Okta requires the `firstName` and `lastName` attributes for a user. Include the `name` scope if the the Identity Provider needs to support JIT.
```

Comment:
Apple does support the email attribute, so we might remove `If your Identity Provider doesn't support this attribute, you can make it optional. See [Manage User Profiles](https://help.okta.com/en/prod/Content/Topics/Directory/eu-profile-editor.htm).`.

`For Just In Time (JIT) provisioning, Okta requires` might change to For Just In Time (JIT) provisioning, Okta **also** requires, since it requires email as well.